plugins/sound/msd-sound-manager.c

```diff
@@ -351,21 +353,14 @@ msd_sound_manager_class_init (MsdSoundManagerClass *klass)
 static void
 msd_sound_manager_init (MsdSoundManager *manager)
 {
-        manager->priv = msd_sound_manager_get_instance_private (manager);
```

Comment:
Same as in typing-break.
https://travis-ci.org/github/mate-desktop/mate-settings-daemon/jobs/758657309#L3197
```
msd-sound-manager.c:354:42: warning: unused parameter 'manager' [-Wunused-parameter]

msd_sound_manager_init (MsdSoundManager *manager)
```
I guess we have to live with that warning, or not?

Reply:
Yes, the `instance_init` function is mandatory and it follows a function signature pattern.
https://developer.gnome.org/gobject/stable/howto-gobject-construction.html